### PR TITLE
Add isFullWidth and size options to Button

### PR DESCRIPTION
**Pull request by ldabiralai**

### CODE REVIEW

**Comment by rogerfranklin2** [comment]
https://deploy-preview-12--eurosport.netlify.com/

Looks like netlify problems?

**Comment by ldabiralai** [comment]
Try: https://deploy-preview-12--es-web-toolkit.netlify.com as we changed the netlify app name

**Comment by bdo** [comment]
👍 to @rogerfranklin2's suggestion

File: src/atoms/Button/index.js [hunk 1]
```diff
@@ -44,10 +44,14 @@ const Button = styled(Link)`
 
 Button.defaultProps = {
   type: 'primary',
+  size: 'default',
+  isFullWidth: false,
```
Comment (JoffreyMaheo) [inline src/atoms/Button/index.js]: I think the button can take by default the full width, in fact in our react page, the button always take the full width, is it the same for you ?

Comment (bdo) [reply src/atoms/Button/index.js]: How about `isAutoWidth` which defaults to false? Then we only have to specify it in the Footer and by default it’s full width.

**Comment by ldabiralai** [comment]
Closing this as we agreed to handle the sizing of the button within the applications until the footer moves into here.

**Pull request closed without merging**